src/client/liveblog.ts

```diff
@@ -42,7 +42,7 @@ fetch(`${window.location}/live-blocks`)
     .then(({ newBlocks }) => {
         const blocks = deserialise(newBlocks);
 
-        ReactDOM.render(
+        ReactDOM.hydrate(
```

Comment:
Heads up that, as discussed [here](https://github.com/guardian/apps-rendering/pull/345#discussion_r430994045), there's no guarantee that the data matches what was available on the server, which may result in rendering bugs when using `hydrate`.